src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,22 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, body, user, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+      <p>
+        {' Posted by '}
+        <UserInfo user={user} />
```

Comment:
user can be null, so it's better to add the check
```suggestion
        {user && <UserInfo user={user} />}
```